tests/schedulers/utilization_scheduler_test.py

```diff
@@ -29,28 +34,44 @@ def __call__(self, *args, **kwargs):
         return self.now
 
 
+@pytest.mark.usefixtures("reset_recorded_times")
 def test_invalid_target_utilization():
     with pytest.raises(ValueError) as ex:
         UtilizationBasedScheduler(params={
             "target-utilization": 200.432,
-            "warmup-time-period": 100
+            "record-response-times": False
         })
 
     assert "target-utilization must be in the range (0.0, 1.0] but is 200.432" == str(ex.value)
 
     with pytest.raises(ValueError) as ex:
         UtilizationBasedScheduler(params={
             "target-utilization": 0.0,
-            "warmup-time-period": 100
+            "record-response-times": False
         })
 
     assert "target-utilization must be in the range (0.0, 1.0] but is 0.0" == str(ex.value)
 
 
+@pytest.mark.usefixtures("reset_recorded_times")
```

Comment:
If you want to avoid repeatedly calling `@pytest.mark.usefixtures` all these can be placed under a `class TestScheduler` and only the first method needs to be marked with `@pytest.mark.usefixtures("reset_recorded_times", autouse=True)`, see https://docs.pytest.org/en/latest/unittest.html#using-autouse-fixtures-and-accessing-other-fixtures

Reply:
As discussed offline let's skip this for now.